source/settings/plugin/compress-plugin.rst

```diff
@@ -15,3 +15,23 @@ Configuration:
   protocol imap {
     mail_plugins = $mail_plugins imap_zlib
   }
+
+  plugin {
+    ## imap_zlib_<algorithm>_compress_level = value
+    ##
+    # imap_zlib_deflate_compress_level = 6
```

Comment:
This is now describing only the new setting, without even mentioning the old or versionadded/changed.